src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Random random = new Random();
+        int randomColor = random.nextInt(Color.values().length);
+        return Color.values()[randomColor].name();
```

Comment:
random can be a class field to reuse it,
`Color.values()` can be a method variable to avoid multiple calls